share/dkg/pedersen/structs.go

```diff
@@ -292,6 +331,6 @@ func VerifyPacketSignature(c *Config, p Packet) error {
 		return errors.New("unknown packet type")
 	}
 
-	err := c.Auth.Verify(pub, hash, sig)
+	err = c.Auth.Verify(pub, hash, sig)
 	return err
```

Comment:
See below my other comments about returning errors in verification methods.
But also
```suggestion
	return c.Auth.Verify(pub, hash, sig)
```